### PR TITLE
Demystifying geospatial data

### PR DESCRIPTION
**Pull request by drelliche**
High level introduction to geospatial data, benefits and issues of using it.

Looking for feedback particularly from those with more experience in the like @pm0kjp

### CODE REVIEW

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:121]:
```suggestion
The quantity of data and how to determine which geographic data you need is another important consideration. The abundance of geospatial data created by modern systems also creates new problems. While it might seem like more data is always better, even the most high powered computer can only handle so much data at a time.
```

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:125]:
```suggestion
Of course there is far more location data than that collected by satellites.  Environmental sensors, sales records, bus routes, census data, school district boundaries and many more sources provide rich location data that might be useful to you!
```

Or something like this?

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:142]: Consider a caption with "used with permission from..." (and ask for permission or say it's fair use etc)

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:152]: This is very brief -- can you enrich, or roll into an earlier page, so it's not so much whitespace on the page?

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:168]:
```suggestion
Location information doesn't have to be as precise as an address to be useful; knowing what neighborhood study participants live in is definitely geospatial data. The location of public amenities like swimming pools is also geospatial data.
```

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:175]:
```suggestion
Which of these are potential benefits of incorporating geospatial data into public health research? Select all that apply.
```

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:193]:
```suggestion
[[X]] The privacy of study participants' addresses and other location data must be carefully protected.
```

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (pm0kjp) [inline demystifying_geospatial_data/demystifying_geospatial_data.md:203]: Add additional resources!

**Review by pm0kjp — CHANGES_REQUESTED** [review]
May add some other stuff as I go through the checklist but I didn't want to make you wait any longer!

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:85]: I have no objections to title case, and since you prefer it, I'll go with it everywhere :)

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:119]: done!

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:106]: done!

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:89]: done

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:95]: Thanks! fixed

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:53]: I was holding this for the next module, but I agree it should go here!

Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:53]: added!

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:153]: folded in as a subsection of another page

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:53]: @pm0kjp would it make sense to have a small map here (thinking Roberts building as a point, South St as a line segment, and 19146 as a polygon)? I want to make such an image for the other module I'm working on focusing on these types of data, so it might make sense to put a visual here?

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:79]: I've moved this section up in the module, but I could definitely use another set of eyes on its current location @paytonk !

I think now that it is earlier in the module, these things are all addressed later on in it so adding them here wouldn't make sense, but I'm definitely ready and open to pushback on that!

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:91]: I think I have now incorporated this, but will need another look through to make sure things flow.

**Comment by drelliche** [comment]
@drelliche Note to self: give the whole thing an end to end read before you add a bunch more links, create a new image, and send it back for re-review.

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:168]: Having done some more copyright research... this image (and article) are published by Elsevier and they, not the authors, are the copyright holders. I have no interest in interacting with Elsevier, so I am removing the image.

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:230]: oops! done!

**Comment by drelliche** [comment]
@pm0kjp Thanks for your excellent comments and suggestions. I've made some pretty big changes, and the flow is both quite different now, and (I think) better. When you have a few minutes to do another read-through, I'd especially appreciate you thoughts on the following:
- Does it read through as a coherent document?
- Should I add a map on page 2 before the "more than maps" subsection
- Are there other privacy techniques I should be mentioning? 
- Does the "jittering" section still make sense without the map? Especially if I make a map for page 2, I could make a map for this section.

**Comment by pm0kjp** [comment]
@drelliche I am woefully remiss in looking this over, will review ASAP and either merge the PR or make new suggestions.  TYSM!

**Review by pm0kjp — CHANGES_REQUESTED** [review]
Just a couple small things

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:18]: "Define" works for me!

File: demystifying_geospatial_data/demystifying_geospatial_data.md
Comment (drelliche) [reply demystifying_geospatial_data/demystifying_geospatial_data.md:55]: "latitutde" lol

**Review by pm0kjp — APPROVED** [review]

**Pull request merged**
